src/scripts/main.js

```diff
@@ -1,3 +1,30 @@
 'use strict';
 
-// write your code here
+const promise1 = new Promise((resolve) => {
+  document.querySelector('.logo').addEventListener('click', () => {
+    resolve();
+  });
+});
+
+const promise2 = new Promise((resolve, reject) => {
+  setTimeout(() => reject(new Error('Timeout occurred')), 3000);
+});
+
+function handleSuccess() {
+  const div = document.createElement('div');
+
+  div.classList.add('message');
+  div.textContent = 'Promise was resolved!';
+  document.body.appendChild(div);
+}
+
+function handleError() {
+  const div = document.createElement('div');
+
+  div.classList.add('message', 'error-message');
+  div.textContent = 'Promise was rejected!';
+  document.body.appendChild(div);
+}
+
+promise1.then(handleSuccess).catch(handleError);
+promise2.then(handleSuccess).catch(handleError);
```

Comment:
Currently, `promise1` and `promise2` are handled independently. If you want to handle them together, you might consider using `Promise.race` or `Promise.all` depending on your requirements. `Promise.race([promise1, promise2])` will resolve or reject as soon as one of the promises resolves or rejects, while `Promise.all([promise1, promise2])` will wait for both promises to settle.